.github/CODEOWNERS

```diff
@@ -568,7 +568,6 @@ src/platform/packages/shared/kbn-rule-data-utils @elastic/security-detections-re
 src/platform/packages/shared/kbn-safer-lodash-set @elastic/kibana-security
 src/platform/packages/shared/kbn-saved-search-component @elastic/obs-ux-logs-team
 src/platform/packages/shared/kbn-scout @elastic/appex-qa
-src/platform/packages/shared/kbn-scout-release-testing @elastic/appex-qa
```

Comment:
since we back to the topic: shouldn't we make this package `private`? What are the chances it will be imported and re-used?